### PR TITLE
Responsive event website

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
In this PR:

I made a responsive website for events.

### CODE REVIEW

File: aboutpage.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    </div>
+  </section>
+  <footer class="footer">
+    <img src="./images/RO.jpeg" alt="" class="footer-img" />
+    <p class="footer-para">
+      2023 Made in Rwanda Exhibition. <br />Some Rights Reserved.
+    </p>
+  </footer>
+
+    <script src="./aboutpage.js"></script>
```
Comment (Shedrack-Sunday) [inline aboutpage.html]: - [ ] Nice work you did here. however, the background on the desktop should be dark based on the figma design provided. I've attached a photo comparison to show you the difference. sticking to the design  is important as that is how potential clients or your future managers will ask for 👏 

# Your view:
image


# Figma:
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      <h3 id="p-title">Partners</h3>
+      <hr class="line" />
+      <div class="partners-c">
+        <img id="wood" src="./images/Logo 10.png" alt="Wood and words" />
+        <img id="toddle-care" src="./images/tc.jpg" alt="Toddle care" />
+      </div>
+    </div>
+
+    <footer class="footer">
+      <img src="./images/RO.jpeg" alt="" class="footer-img" />
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Its best practice to include an `alt` on all images. please add one here to make your work professional 👏

File: aboutpage.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            <li>My page</li>
+            <li>Log out</li>
+          </ul>
+        </div>
+
+        <div class="all">
+           <nav class="nav-bar">
+            <a class="nav-img" href="./images/RO.jpeg"><img id="nav-img" src="./images/RO.jpeg" alt=""></a>
+            <ul class="navigation">
+              <li class="nav-item"><a href="./index.html" class="nav-link">Home</a></li>
```
Comment (Shedrack-Sunday) [inline aboutpage.html]: - [ ] Please always remember to include a descriptive meaning to the image using the `alt`. i made a detailed review on the summary, please check it out 👏

File: homepage.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+];
+
+const speakers = document.querySelector('.speaker');
+speaker.forEach((speak) => {
+  const divElem = document.createElement('div');
+  divElem.innerHTML = `
+    <div class="home-speaker ${speak.display}">
+    <div class="speaker-img">
+    <img src=${speak.img} alt="">
+    </div>
```
Comment (Shedrack-Sunday) [inline homepage.js:76]: - [ ] Please always remember to include a descriptive meaning to the image using the `alt`. i made a detailed review on the summary, please check it out 👏

File: index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+              ></a>
+            </li>
+            <li>English</li>
+            <li>My page</li>
+            <li>Log out</li>
+          </ul>
+        </div>
+<div class="all">
+          <nav class="nav-bar">
+            <a class="nav-img" href="./index.html"><img id="nav-img" src="./images/RO.jpeg" alt=""></a>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Here also, remember to include a descriptive meaning to the image using the `alt`. i made a detailed review of the summary, please check it out 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Roland-Ntwali  👋

CHANGES REQUIRED ♻️
Great work on the project 👏
You've done well implementing the requirements, However, there are a few you need to work on, to move to the next project. 💯


Highlights
✅All linters are checked 🔢 
✅ Descriptive commit messages 👍 
✅ Nice design 🥇 
✅ Nice use of javascript 💯 

Required Changes ♻️

 - [ ] You did make effort in the Project and you really did a great job  👏 . However, there seem to be a few adjustments needed to be made, based on the design given, to make your whole code stand out and look good. 👍

-  [ ] Secondly, you missed some alternate words for `img` tag. The alt attribute in CSS is used to provide alternative text for images. It's an important aspect of web accessibility, as it helps visually impaired users understand the content of images through screen readers and other assistive technologies.

##  Here are a few reasons why the alt attribute is important in CSS:

- Accessibility: As mentioned, the alt attribute is critical for making the web accessible to people with visual impairments. Screen readers can read out the alternative text, allowing visually impaired users to understand the content of an image.

- Search Engine Optimization (SEO): The alternative text also helps search engines understand the content of an image, which can improve the ranking of a webpage in search results.

- Fallback Content: If an image fails to load, the alternative text will be displayed instead, ensuring that the content of the page remains accessible even if images don't load properly.

- In summary, the alt attribute is an essential part of CSS and web development, as it helps to make the web more accessible and user-friendly, while also improving search engine rankings.




 Kindly fix the margin on the right of the screen view. good job so far 👏

check the comment section for a detailed review

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by andgarzonmal — APPROVED** [review]
Hi @Roland-Ntwali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**